src/GovUk.Education.ExploreEducationStatistics.Admin/Services/FileRepository.cs

```diff
@@ -20,16 +20,51 @@ public FileRepository(ContentDbContext contentDbContext)
             _contentDbContext = contentDbContext;
         }
 
-        public async Task<File> Create(Guid releaseId,
+        public async Task<File> CreateAncillaryOrChart(
```

Comment:
For the time being these methods will do, but long term it would be good to refactor this into something cleaner (a potential example I've previously provided was: https://github.com/dfe-analytical-services/explore-education-statistics/pull/2116#discussion_r515893315)